CHAINMAIL.md

```diff
@@ -0,0 +1,36 @@
+Part of the Agoric strategy for providing security is based on the
+[Formal Methods](https://agoric.com/papers/#formal-reasoning) literature, which has developed tools
+for proving that programs follow their specifications. There have been interesting advances in this
+area including the development of [seL4](https://sel4.systems/Info/FAQ/proof.pml) and the
+[proof of its correctness](https://sel4.systems/Info/FAQ/proof.pml). We have been collaborating with
+a research group at Victoria University who are investigating how to apply these techniques to
```

Comment:
The group is actually split between Imperial (Sophia and Susan), Victoria (James), and Data61 (Toby). See the attributions at the top of https://ai.google/research/pubs/pub44272.pdf but add Susan Eisenbach of Imperial

Reply:
I just listed institutions. I gave UNSW for Toby. Would individual names be better?

Reply:
> Would individual names be better?

yes

Reply:
done